lib/main.dart

```diff
@@ -128,8 +126,7 @@ class _MyHomePageState extends State<MyHomePage> with WidgetsBindingObserver {
     }
   }
 
-  Future<dynamic> subscribeToDevive(List<int?> deviceIds) async =>
-      await hubConnection.invoke("Subscribe", args: [deviceIds]);
+  Future<dynamic> subscribeToDevive(List<int?> deviceIds) async => await hubConnection.invoke("Subscribe", args: [deviceIds]);

```

Comment:
It seems like your editor settings are incorrect, because this shouldn't be in a single line. Line length should only be 120 chars, but is 127 with this